config/application.rb

```diff
@@ -127,8 +127,7 @@ class Application < Rails::Application
     # like if you have constraints or database-specific column types
     # config.active_record.schema_format = :sql
 
-    # enables in memory cache store with ttl
-    #config.cache_store = TimedCachedStore.new
+    # By default use file store
     config.cache_store = :file_store, Rails.root.join("tmp", "cache")
```

Comment:
I suppose the problem would go away if we cached these fragments in memory too. After the upgrade or plugin installation the cache would have to be regenerated.